roles/telemetry_autoscaling/tasks/test_autoscaling.yml

```diff
@@ -57,7 +57,7 @@
   register: busy_process
   with_items: "{{ vnf_instance_ip.stdout_lines }}"
 
-- name: Verify that the alarm has been triggered
+- name: RHOSO-12660 Verify that the alarm has been triggered
   ansible.builtin.shell: |
     # source ~/overcloudrc;
     {{ openstack_cmd }} alarm list -c state -c name -f value| \
```

Comment:
This could probably be more robust. If there is another stack present, then this will fail. It might need investigating.